OpenSim/Auxiliary/auxiliaryTestFunctions.h

```diff
@@ -79,16 +79,16 @@ inline void ASSERT(bool cond,
  */
 void CHECK_STORAGE_AGAINST_STANDARD(OpenSim::Storage& result, 
                                     OpenSim::Storage& standard, 
-                                    OpenSim::Array<double> tolerances, 
+                                    std::vector<double> tolerances, 
                                     std::string testFile, 
                                     int testFileLine, 
                                     std::string errorMessage)
 {
```

Comment:
@klshrinidhi is not suggesting to make this templated. He is just using `T` as a placeholder for `std::vector<double>`, `std::string`, etc.